modules/ROOT/nav.adoc

```diff
@@ -3,19 +3,20 @@ include::{partialsdir}/attributes.adoc[]
 * xref:getting-started.adoc[Getting Started]
 ** xref:getting-started.adoc#introduction[Introduction]
 ** xref:getting-started.adoc#setting-up-aerogear-mobile-services-on-openshift[Setting Up]
+** xref:getting-started.adoc#provision-mdc[Provisioning Mobile Deveoper Console]
 ** xref:getting-started.adoc#registering[Registering a Mobile Client]
 ** xref:getting-started.adoc#provisioning[Provisioning a Service]
 ** xref:getting-started.adoc#binding[Binding Services]
 ** xref:getting-started.adoc#setting-up-your-local-development-environment[Setting Up Development Environment]
 ** xref:getting-started.adoc#running[Running a Mobile App]
 
-* xref:mobile-clients.adoc[Mobile Clients]
-** xref:mobile-clients.adoc#introduction[Introduction]
-** xref:mobile-clients.adoc#registering[Registering a Mobile Client]
-** xref:mobile-clients.adoc#services[Viewing Available Services]
-** xref:mobile-clients.adoc#building[Building Mobile Apps]
-** xref:mobile-clients.adoc#sdk[AeroGear SDK]
 
+* xref:mobile-developer-console.adoc[Mobile Developer Console]
+** xref:mobile-developer-console.adoc#introduction[Introduction]
+** xref:mobile-developer-console.adoc#registering[Registering a Mobile Client]
+*** xref:mobile-developer-console.adoc#sdk[Configure SDK]
+** xref:mobile-developer-console.adoc#services[Viewing Available Services]
+** xref:mobile-developer-console.adoc#building[Building Mobile Apps]
```

Comment:
Should we remove this section? Builds won't be productized.

Reply:
@jhellar we'll still support it upstream so I am leaving it in here. We we wont show these docs downstream.